sdk/compute/armcompute/go.mod

```diff
@@ -7,3 +7,6 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.19.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.10.0
 )
+
+// To better align with the Azure SDK guidelines (https://azure.github.io/azure-sdk/general_introduction.html), we have decided to change the module path to "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute". Therefore, we are retracting the old module path (which is "github.com/Azure/azure-sdk-for-go/sdk/compute/armcompute") to avoid confusion.
+retract [v0.0.1, v0.3.1]
```

Comment:
We want to deprecate these modules, not retract them.  See https://tip.golang.org/ref/mod#go-mod-file-module-deprecation

Reply:
Yes, that's true

Reply:
Retract comment and changelog changed.